app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -231,18 +263,29 @@ public void showScanButton(boolean show) {
     public void showAttendees(List<Attendee> attendees) {
         // The list is loaded from presenter, so we just need
         // to notify RecyclerView to update the data
-        itemAdapter.set(attendees);
+        itemAdapter.setNewList(attendees, true);
         binding.setVariable(BR.attendees, attendees);
         binding.executePendingBindings();
     }
 
     @Override
     public void updateAttendee(int position, Attendee attendee) {
-        itemAdapter.set(position, attendee);
+        itemAdapter.setNewList(attendeesPresenter.getAttendees(), true);
```

Comment:
Its is required to manage correct item update.
Thats where I was working. Just using `set` was giving indexbound exception in filter case.
Hence passing the reference of list again is only option found.
I tested it with 1000+ attendees. Working fine.